### PR TITLE
Add watchlist mutations

### PR DESCRIPTION
**Pull request by maybelinot**
@flashlee @alienalien13  review pls

### CODE REVIEW

File: src/resolvers.ts [hunk 1]
```diff
@@ -16,4 +20,18 @@ export const resolvers = {
       return context.dataSources.theMovieDB.getMoviesByTitle(args.title)
     }
   },
+  Mutation: {
+    addToWatchList: (root: any, args: any, context: any) => {
+      context.db().collection('poc').doc('watchList').update({
+        list: context.db.FieldValue.arrayUnion(args.movieID)
```
Comment (flashlee) [inline src/resolvers.ts]: Can you please explain this line for me? )

File: src/index.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ const serviceAccount = getServiceAccount()
 // auth to firebase
 admin.initializeApp({ credential: admin.credential.cert(serviceAccount) })
 // firestore DB ref
-var db = admin.firestore()
+var db = admin.firestore
```
[the remark this answers is not in the document]
Comment (maybelinot) [reply src/index.ts]: to use admin context helper functions within resolver

File: src/resolvers.ts (hunk 1 above)
Comment (maybelinot) [reply src/resolvers.ts]: that's special helper functions provided by [firestore](https://firebase.google.com/docs/reference/js/firebase.firestore.FieldValue) to mutate their data structure.

File: src/resolvers.ts [hunk 3]
```diff
@@ -1,9 +1,13 @@
 export const resolvers = {
   Query: {
     watchlist: (root: any, args: any, context: any) => {
-      return context.db.collection('poc').get()
-      .then((snapshot: any) => {
-        return snapshot.docs[0].data().watchList
+      return context.db().collection('poc').doc('watchList').get()
+      .then((doc: any) => {
+        if (!doc.exists) {
+          console.log("Document doesn't exist")
```
[the remark this answers is not in the document]
Comment (flashlee) [reply src/resolvers.ts]: Probably you are right. The difference for `console.log` and `console.info` can be noticed only in browser. For node.js both make output to `stdout`

[the remark this answers is not in the document]
Comment (flashlee) [reply src/resolvers.ts]: the only diference can be here is in code aesthetics. We want to log this message or we want to inform that document doesn't exist? :)

**Pull request merged**